tarofreighter/chain_porter.go

```diff
@@ -930,6 +930,11 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 			return nil, err
 		}
 
+		chainFees, err := tarogarden.GetTxFee(currentPkg.SendPkt)
+		if err != nil {
+			return nil, err
```

Comment:
nit: add context to error and wrap it? I think we should start doing that in our projects to always get an "error trace" message. Same in the caretaker.